src/tasks/minions/groupMonsterActivity.ts

```diff
@@ -16,14 +16,16 @@ export default class extends Task {
 		const teamsLoot: { [key: string]: ItemBank } = {};
 		const kcAmounts: { [key: string]: number } = {};
 
+		for (let j = 0; j < users.length; j++) {
+			kcAmounts[users[j]] = 0;
+		}
+
 		for (let i = 0; i < quantity; i++) {
 			const loot = monster.table.kill(1);
 			const userWhoGetsLoot = randomItemFromArray(users);
 			const currentLoot = teamsLoot[userWhoGetsLoot];
 			teamsLoot[userWhoGetsLoot] = addBankToBank(currentLoot ?? {}, loot);
-			kcAmounts[userWhoGetsLoot] = Boolean(kcAmounts[userWhoGetsLoot])
-				? kcAmounts[userWhoGetsLoot]++
```

Comment:
could you try `? ++kcAmounts[userWhoGetsLoot]` or alternatively `? kcAmounts[userWhoGetsLoot] + 1`
instead of the current commit.  The problem seems to be the post-increment operator returning the original value instead of the modified one

Reply:
Both worked but I changed it to `? ++kcAmounts[userWhoGetsLoot]` as it was cleaner.